apps/web/src/serlo-editor-integration/create-plugins.tsx

```diff
@@ -84,6 +85,12 @@ export function createPlugins({
       visibleInSuggestions: true,
       icon: <IconMultimedia />,
     },
+    {
+      type: EditorPluginType.DragDropBg,
+      plugin: createDragDropBgPlugin(),
+      visibleInSuggestions: true,
```

Comment:
always good to do it now :) 
```suggestion
      visibleInSuggestions: !isProduction,
```

Reply:
Done :heavy_check_mark: